src/resources/defi/types.ts

```diff
@@ -59,44 +71,70 @@ export type Borrow = {
   quantity: string;
   total_asset: string; // what does this mean?
   omit_from_total?: boolean;
-  underlying: { asset: PositionAsset; quantity: string }[];
+  underlying: UnderlyingAsset[];
+};
+export type Stake = {
+  asset: PositionAsset;
+  quantity: string;
+  apr?: string;
+  apy?: string;
+  total_asset?: string; // what does this mean?
```

Comment:
couple of these comments here that you left, did you end up figuring this out?

Reply:
those comments were there before, but I also wasn't sure what that field was supposed to represent

Reply:
ahh okay feel free to ignore then